src/Utopia/Messaging/Adapters/SMS/Termii.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+// Reference Material
+// https://developers.termii.com/messaging-api
+class Termii extends SMSAdapter
+{
+    const SMSType = 'plain';
+
+    const SMSChannel = 'generic';
+
+    public function __construct(
+        private string $apiKey
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return 'Termii';
+    }
+
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 100;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Exception
+     */
+    protected function process(SMS $message): string
+    {
+        return $this->request(
+            method: 'POST',
+            url: 'https://api.ng.termii.com/api/sms/send',
+            headers: [
+                'Content-Type: application/json',
+            ],
+            body: \json_encode(
+                $this->getRequestBody(
+                    to: $message->getTo(),
+                    text: $message->getContent(),
+                    from: $message->getFrom()
+                )
+            ),
+        );
+    }
+
+    /**
+     * Get the request body
+     *
+     * @param  array  $to  Phone number
+     * @param  string  $text  Message to send
+     * @param  string|null  $from Origin of the message
+     */
+    private function getRequestBody(array $to, string $text, string $from = null): array
+    {
+        $from = $from ?? '';
```

Comment:
It's okay to pass an empty string for from?

Reply:
No, it will throw exception as Invalid Sender Id

Reply:
And what if null is passed?

Reply:
Hi, sorry for late reply. I was busy on some office work.
If null/empty from passed then also it will throw exception

Exception: {"message":"The given data was invalid.","errors":{"from":["The from field is required."]}}